### DISCUSSION TITLE
Did someone manage to run a mongoDB container with azure files persistence?

### DISCUSSION THREAD

**Discussion by rhuanbarreto** [microsoft/azure-container-apps#702] [Q&A]
I've tried to run a mongoDB container as a sidecar and persist its storage to azure files volume mount without success. The container doesn't even emit any logs about it. Is there any incompatibility with azure files?

**Comment by Sarmen91** [comment]
Any news ?

**Reply by rhuanbarreto** [reply]
No. Don't think they support this even with NFS file shares.
